src/library/crate/cratestorage.cpp

```diff
@@ -453,6 +453,23 @@ CrateTrackSelectResult CrateStorage::selectTrackCratesSorted(TrackId trackId) co
     }
 }
 
+CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QString& crateNameLike) const {
```

Comment:
This function should be responsible for the escaping. The caller shouldn't need to worry about that detail.